src/statement.cc

```diff
@@ -226,7 +226,7 @@ template <class T> T* Statement::Bind(const Napi::CallbackInfo& info, int start,
     Napi::Env env = info.Env();
     Napi::HandleScope scope(env);
 
-    if (last < 0) last = info.Length();
+    if (last < 0) last = static_cast<int>(info.Length());
```

Comment:
Why not change the sig to `Statement::Bind(const Napi::CallbackInfo& info, size_t start, size_t last) ` ?

Reply:
The sig currently has start and last as default parameters with last set to -1 by default. I can change the sig, though.

Reply:
The only place I see it get called is here https://github.com/microsoft/vscode-node-sqlite3/blob/a6204e89f7be6da3a7602560ec410b51a6957a47/src/statement.cc#L630 were last value is `info.length()` by default, so I am sure why this can be `-1` ? If there are no other call sites then I suggest to change the sig.

Reply:
Lets change the sig and have the callers explicitly pass in start and last.